eth/catalyst/api.go

```diff
@@ -1163,6 +1113,23 @@ func (api *ConsensusAPI) heartbeat() {
 	}
 }
 
+// config retrieves the chain's fork configuration.
+func (api *ConsensusAPI) config() *params.ChainConfig {
+	return api.eth.BlockChain().Config()
+}
+
+// checkFork returns true if the latest fork at the given timestamp
+// is one of the forks provided.
+func (api *ConsensusAPI) checkFork(timestamp uint64, forks ...forks.Fork) bool {
```

Comment:
naming is a bit ambiguous on its face.  Could mean "check that the forks are activated".

But the fact that this takes multiple fork values and we ensure ensure that a fork cannot be activated unless all past ones are, clears up this ambiguity.

Reply:
yeah i couldn't come up with a name i really liked here, originally it was `isLatestFork` which wasn't my favorite either